Sources/TuistGenerator/Generator/SchemesGenerator.swift

```diff
@@ -437,4 +441,18 @@ final class SchemesGenerator: SchemesGenerating {
         }
         return path
     }
+
+    private func defaultDebugBuildConfigurationName(in project: Project) -> String {
+        let debugConfiguration = project.settings.defaultDebugBuildConfiguration()
+        let buildConfiguration = debugConfiguration ?? project.settings.configurations.keys.first
+
+        return buildConfiguration?.name ?? "Debug"
```

Comment:
just a minor thing... Can't we use `.debug` from `BuildConfiguration .Variant` instead of using the string directly? Just to prevent typos in the configuration in the future.